src/test/java/seedu/saveit/logic/commands/SelectCommandTest.java

```diff
@@ -36,21 +36,22 @@
     private CommandHistory commandHistory = new CommandHistory();
 
     @Test
-    @Ignore
     public void execute_validIndexUnfilteredList_success() {
         Index lastIssueIndex = Index.fromOneBased(model.getFilteredAndSortedIssueList().size());
 
         assertExecutionSuccess(INDEX_FIRST_ISSUE);
+        model.resetDirectory(new Directory(0, 0));
```

Comment:
I think now this can be replaced by `new DirectoryBuilder().build()`, so that 0 won't appear as magic number

Reply:
will do thanks!